[BUG] Janus module fails to compile

The module fails to compile on arm32, see https://github.com/OpenSIPS/opensips/actions/runs/13672544118/job/38225764545

```
/usr/lib/ccache/gcc -fPIC -DPIC -g -Werror -Werror -DMOD_NAME='janus' -DUNIT_TESTS -fPIE -fPIC -DPKG_MALLOC  -DSHM_MMAP  -DUSE_MCAST  -DDISABLE_NAGLE  -DSTATISTICS  -DHAVE_RESOLV_RES  -DF_MALLOC  -DQ_MALLOC  -DHP_MALLOC  -DDBG_MALLOC  -I/home/runner/work/opensips/opensips/lib/libtap -DHAVE_STDATOMIC -DHAVE_GENERICS   -DNAME='"opensips"' -DVERSION='"3.6.0-dev"' -DARCH='"arm7"' -DOS='"linux"' -DCOMPILER='"/usr/lib/ccache/gcc 13"' -D__CPU_arm7 -D__OS_linux -D__SMP_yes -DCFG_DIR='"/usr/local//etc/opensips/"'  -DVERSIONTYPE='"git"' -DTHISREVISION='"unknown"' -DFAST_LOCK -DADAPTIVE_WAIT -DADAPTIVE_WAIT_LOOPS=1024 -DHAVE_GETHOSTBYNAME2 -DHAVE_UNION_SEMUN -DHAVE_MSG_NOSIGNAL -DHAVE_MSGHDR_MSG_CONTROL -DHAVE_ALLOCA_H -DHAVE_TIMEGM -DHAVE_EPOLL -DHAVE_SIGIO_RT -DHAVE_SELECT -DUNIT_TESTS -Wno-deprecated-declarations -c janus_mod.c -o janus_mod.o
In file included from ../../parser/../ip_addr.h:46,
                 from ../../parser/msg_parser.h:49,
                 from ../../sr_module.h:48,
                 from janus_mod.c:31:
janus_mod.c: In function 'w_janus_send_request':
../../parser/../dprint.h:71:36: error: format '%lu' expects argument of type 'long unsigned int', but argument 12 has type 'uint64_t' {aka 'long long unsigned int'} [-Werror=format=]
   71 |         #define DP_PREFIX  (char *)"%s [%d] "
      |                                    ^~~~~~~~~~
../../parser/../dprint.h:456:41: note: in definition of macro 'MY_DPRINT'
  456 |                                         _stderr_prefix LOG_PREFIX _fmt, \
      |                                         ^~~~~~~~~~~~~~
../../parser/../dprint.h:96:26: note: in expansion of macro 'DP_PREFIX'
   96 | #define DP_DBG_PREFIX    DP_PREFIX "%s" DP_DBG_TEXT
      |                          ^~~~~~~~~
../../parser/../dprint.h:602:80: note: in expansion of macro 'DP_DBG_PREFIX'
  602 |                                                 MY_DPRINT(L_DBG, log_facility, DP_DBG_PREFIX, \
      |                                                                                ^~~~~~~~~~~~~
janus_mod.c:232:9: note: in expansion of macro 'LM_DBG'
  232 |         LM_DBG("Expecting reply for transaction %lu\n",reply_id);
      |         ^~~~~~
cc1: all warnings being treated as errors
make[2]: *** [../../Makefile.rules:29: janus_mod.o] Error 1
ERROR: Building janus module failed!
make[1]: *** [Makefile:189: modules/janus] Error 2
make[1]: Leaving directory '/home/runner/work/opensips/opensips'
make: *** [Makefile:210: modules] Error 2
```

--- a/modules/janus/janus_mod.c
+++ b/modules/janus/janus_mod.c
@@ -229,7 +229,7 @@ static int w_janus_send_request(struct sip_msg *msg, str *janus_id,str *request,
 		return 1;
 	}
 
-	LM_DBG("Expecting reply for transaction %lu\n",reply_id);
+	LM_DBG("Expecting reply for transaction %lu\n", (unsigned long)reply_id);
 
 	for (total_us = 0; total_us < janus_cmd_timeout * 1000;
 	     total_us += janus_cmd_polling_itv) {

